### PR TITLE
Add ETL pipeline testing QS (e2e-python)

### PR DESCRIPTION
**Pull request by cristianhkr**
Provide a new end to end testing quickstarter to test ETLs pipelines.

Technologies:
-pytest
-greatexpectations
-allure
-boto3
...

### CODE REVIEW

**Comment by metmajer** [comment]
@braisvq1996 @cristianhkr not sure this QS is appealing to an open source audience. Please move it to GitHub.com/Boehringer-Ingelheim.

**Comment by bassagan** [comment]
> @braisvq1996 @cristianhkr not sure this QS is appealing to an open-source audience. Please move it to GitHub.com/Boehringer-Ingelheim.

This QS does not need any infrastructure besides an AWS account and is not related to anything specific to Boehringer. This is a testing framework that executes its tests on the cloud, so it might fit an open-source community like AWS or Azure QS.   

We can discuss it internally if you consider we should be moving it to the private repository.

**Comment by metmajer** [comment]
@braisvq1996 as discussed with @bassagan, the Quickstarter is generally usable and not only in our enterprise context. Good for me to take this up in our open source library.

File: be-gateway-nginx/files/docker/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # https://github.com/openresty/docker-openresty
-FROM openresty/openresty:1.25.3.1-1-rocky
+FROM openresty/openresty:1.21.4.1-rocky
```
Comment (braisvq1996) [inline be-gateway-nginx/files/docker/Dockerfile:2]: This needs to be updated so the version in master is not downgraded

File: be-gateway-nginx/files/metadata.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: nginx
-description: "Enhanced nginx with Lua embeded. nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Technologies: OpenResty/nginx 1.25.3.1-1"
+description: "Enhanced nginx with Lua embeded. nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Technologies: OpenResty/nginx 1.19.3.2"
```
Comment (braisvq1996) [inline be-gateway-nginx/files/metadata.yml:3]: Update with the content from master branch

File: docs/modules/quickstarters/pages/be-gateway-nginx.adoc [hunk 3]
```diff
@@ -109,9 +109,9 @@ Note that the xref:jenkins-shared-library:labelling.adoc[OpenShift resources wil
 
 ```yaml
 name: nginx
-description: "Enhanced nginx with Lua embeded. nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Technologies: OpenResty/nginx 1.25.3.1-1"
+description: "Enhanced nginx with Lua embeded. nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server. Technologies: OpenResty/nginx 1.19.3.2"
 supplier: https://openresty.org
-version: 1.25.3.1-1
+version: 1.21.4.1
```
Comment (braisvq1996) [inline docs/modules/quickstarters/pages/be-gateway-nginx.adoc:114]: Update with the content from master branch

File: e2e-python/Jenkinsfile.template [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+def generateTerraformOutputsFile() {
+    sh 'terraform output -json > terraform_outputs.json'
+    sh 'cat terraform_outputs.json'
+}
+
+def stageGetNamesFromOutputs() {
+    def outputNames = [:]
+    def terraformOutputJson = readJSON file: 'terraform_outputs.json'
+    //def environmentVarsJson = readJSON file: env.auto.tfvars.json
```
Comment (braisvq1996) [inline e2e-python/Jenkinsfile.template:81]: why this line is commented, if it is mean to be we should add an explanation above

File: release-manager/files/README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ The release manager supports the orchestration of multiple repositories into a l
 
 ### Automated Resolution of Dependencies
```
Comment (braisvq1996) [inline release-manager/files/README.md:17]: File needs to be updated from the content from master branch

**Comment by braisvq1996** [comment]
Quickstarter has been tested and works properly with Release manager process

**Pull request merged**